<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DEVWINDOW</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="start_server">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Start server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_server">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="horizontalWidget_4" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="nickname">
             <property name="text">
              <string>Jason</string>
             </property>
             <property name="placeholderText">
              <string>Nickname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="host">
             <property name="text">
              <string>localhost</string>
             </property>
             <property name="placeholderText">
              <string>Host</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connect_server">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnect_server">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="verticalWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>10</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="bottomMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QPushButton" name="start_game">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Start Game</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cheat">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Cheat</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Sudoqu::GameFrame" name="frame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>1</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">padding: 50px;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="ready">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Ready</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Players :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="player_list">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="chat_area">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>3</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="chat_text">
              <property name="placeholderText">
               <string>Chat</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="chat_send_button">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Sudoqu::GameFrame</class>
   <extends>QFrame</extends>
   <header>gameframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
